src/solution_check.jl

```diff
@@ -0,0 +1,105 @@
+function check_tw_bounds(tws::Vector{Range}, target_time::Float64)
+    
+    for tw in tws
+        if target_time <= tw.ub && target_time >= tw.lb
+            return true
+        end
+    end
+    return false
+
+end
+
+function check_sequence(route::Route, data::RvrpInstance,
+                        computed_data::RvrpComputedData, solution::RvrpSolution,
+                        complete_req_ids::Set{String},
+                        ongoing_req_ids::Set{String})
+
+    for act in route.sequence
+        if act.request_id != "" # Part of a request
+            req = data.requests[computed_data.request_id_2_index[act.request_id]]
+            if req.request_type == 0
+                if act.operation_type == 1
+                    if in(req.id, complete_req_ids)
+                        error("Request ", req.id, " is performed more than once.")
+                    end
```

Comment:
Why is the above test nested. It should be done just after line 19
https://github.com/atoptima/RichVehicleRoutingProblem.jl/pull/32/files#diff-bca8ce6d51b53a49a7c6ff082335b8a7R19

Reply:
I did not understand. What test?